examples/website/src/shared/types.ts

```diff
@@ -157,7 +157,7 @@ export interface Peer {
   websiteURL: string
   themeColor: string
   themeFontColor: string
-  callToActionText: any
+  callToActionText: Node[]
```

Comment:
Why `Node[]`? Was it any before you changed it? This should be `string | Node[]`